packages/im-studio/src/graphql/queries.js

```diff
@@ -1,8 +1,8 @@
 import { gql } from "@apollo/client";
 
 const QUERY_ALL_MOVIES = gql`
-  query {
-    movies {
+  query getAllMovies($userId: String) {
+    movies(userId: $userId) {
       id
```

Comment:
dont use queries inside studio ,use inside imbase
check ott too if the change is not affecting other changes

Reply:
Updated